packages/cozy-sharing/src/components/EditableSharingModal.jsx

```diff
@@ -97,6 +97,7 @@ const contactsQuery = () =>
       ]
     })
     .indexFields(['_id'])
+    .limitBy(1000)
```

Comment:
Can you explain why this change? Does that mean we only fetch the first 100 (now 1000) contacts? Without any pagination? If yes, I don't know if this fix is the right one

Reply:
From my tests, we had a pagination with a limit of 100, so I assumed there is some mechanism somewhere making the pagination?

Reply:
100 is the default behavior from the stack. But we fetchMore by hands to fetch all the contacts here https://github.com/cozy/cozy-libs/blob/master/packages/cozy-sharing/src/components/ContactsAndGroupsDataLoader.jsx . So I assume this modification is good. 

But can you make an explanation in the commit message to say why this change? Since by default we only rely on the first 100 rows returned by the stack? We assume we can get the first 1000 because of the partial index?

Reply:
I set 1000 as a limit to be consistent with [contacts](https://github.com/cozy/cozy-contacts/blob/master/src/helpers/queries.js#L64). And, if 1000 contacts match, I think it's better in this case to have 1 big query of 1000 contacts than 10 of 100 contacts, as we quickly want all the contacts to search on them.
I'll add explanations in the commit.